addons/account/models/product.py

```diff
@@ -34,6 +36,8 @@ class ProductTemplate(models.Model):
         string="Expense Account", oldname="property_account_expense",
         domain=[('deprecated', '=', False)],
         help="The expense is accounted for when a vendor bill is validated, except in anglo-saxon accounting with perpetual inventory valuation in which case the expense (Cost of Goods Sold account) is recognized at the customer invoice validation. If the field is empty, it uses the one defined in the product category.")
+    property_account_income_refund_id = fields.Many2one('account.account', string='Income Refund Account', company_dependent=True,
+        help="If set, this account will be used iby default instead of the income account on the customer credit notes. If not set, the income account will be used")
```

Comment:
spelling iby - also it should be just customer credit notes, not 'the customer credit notes'